api/v1alpha1/rollingupgrade_types.go

```diff
@@ -117,9 +117,14 @@ const (
 )
 
 var (
-	FiniteStates        = []string{StatusComplete, StatusError}
-	AllowedStrategyType = []string{string(RandomUpdateStrategy), string(UniformAcrossAzUpdateStrategy)}
-	AllowedStrategyMode = []string{string(UpdateStrategyModeLazy), string(UpdateStrategyModeEager)}
+	FiniteStates                 = []string{StatusComplete, StatusError}
+	AllowedStrategyType          = []string{string(RandomUpdateStrategy), string(UniformAcrossAzUpdateStrategy)}
+	AllowedStrategyMode          = []string{string(UpdateStrategyModeLazy), string(UpdateStrategyModeEager)}
+	ASGTerminatingInstanceStates = []string{
```

Comment:
nit: can we move this into `aws` package `autoscaling.go` and rename to TerminatingInstanceStates? might be a more appropriate location

Reply:
You are right. Let me do that.